### PR TITLE
Max jj03

### PR DESCRIPTION
**Pull request by MassimoVilla**
BDLaethe33

### CODE REVIEW

File: OxfordBodleian/Juel-Jensen/BDLaethe33.xml [hunk 1]
```diff
@@ -218,39 +258,88 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                            <handNote xml:id="h2" script="Ethiopic">
+                                <locus from="2r" to="130v"/>
+                                <seg type="script"/>
+                                <seg type="ink">Black, red</seg>
+                                <date notBefore="1700" notAfter="1799">18th century</date>
+                                <desc>
+                                    Small and occasionally irregular handwriting.
+                                </desc>
+                                <seg type="rubrication">
+                                    Divine names; number and titles of the psalms; name, number, and traditional interpretation of the Hebrew letter in Ps. 118;
```
Comment (thea-m) [inline OxfordBodleian/Juel-Jensen/BDLaethe33.xml:283]: If I remember correctly, we agreed on "Holy names" in the meeting?

File: OxfordBodleian/Juel-Jensen/BDLaethe33.xml [hunk 2]
```diff
@@ -218,39 +258,88 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
-                                <decoNote xml:id="b1"></decoNote>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <desc type="MagicText"/>  
+                                    <title ref="LIT4625Prayer"/>
+                                    <locus target="#18r"/>
+                                    <note> 
+                                        The note is crudely written in a pale black ink and is incomplete. Many words are not fully understandable. It starts as follows:
+                                        <q xml:lang="gez">
```
Comment (thea-m) [inline OxfordBodleian/Juel-Jensen/BDLaethe33.xml:300]: New additions mark up possibilities! https://betamasaheft.eu/Guidelines/?id=additionsVaria
For incipits, you can use `<seg part="I">` inside `<q>`

File: OxfordBodleian/Juel-Jensen/BDLaethe33.xml [hunk 3]
```diff
@@ -218,39 +258,88 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                                    <q xml:lang="gez">ቤንት፡ ዩውል፡ የንሰን።</q>
+                                    <note>A label with the name of <persName ref="PRS5782JuelJen"/>, written in Ethiopic, is glued onto the inner side of the front board.</note>
+                                </item>
+                                <item xml:id="e2">
+                                    <locus target="#19r #46v #47r"/>
+                                    <note>Scribbles in black ink, partially erased, and pen trials in pencil.</note> 
+                                   </item>
+                                <item xml:id="e3">
+                                    <desc>Psalms are numbered from Ps. 10 onwards.
+                                        A word, effaced at a later stage, was written in correspondence of the midpoint of the Psalms of David, on <locus target="70r"/>.
```
Comment (thea-m) [inline OxfordBodleian/Juel-Jensen/BDLaethe33.xml:316]: `#70r`

File: OxfordBodleian/Juel-Jensen/BDLaethe33.xml [hunk 4]
```diff
@@ -218,39 +258,88 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
-                <!--    <history>
+                <history>
                         <origin>
-                            <origDate when="" evidence=""/>
+                            <origDate from="1700" to="1799" evidence="lettering"/>
                         </origin>
-                        <provenance>  </provenance>
-                    </history>-->
-                    
+                    <provenance> Bought by <persName ref="PRS5782JuelJen"/> at auction on 26 November 1985, according to the Sotheby's sale receipt.</provenance>
```
Comment (thea-m) [inline OxfordBodleian/Juel-Jensen/BDLaethe33.xml:337]: "at an auction"?

**Comment by thea-m** [comment]
This pull request has two very small and easily solvable conflicts to be solved. Would any of you like to do that as a training? If yes and if you need help, let me know!

File: OxfordBodleian/Juel-Jensen/BDLaethe33.xml (hunk 4 above)
Comment (Gnisci) [reply OxfordBodleian/Juel-Jensen/BDLaethe33.xml:337]: I think both are fine.

File: OxfordBodleian/Juel-Jensen/BDLaethe33.xml (hunk 1 above)
Comment (MassimoVilla) [reply OxfordBodleian/Juel-Jensen/BDLaethe33.xml:283]: you're right! It was inherited from a previous description stage and left uncorrected.

**Comment by Gnisci** [comment]
> 
> 
> This pull request has two very small and easily solvable conflicts to be solved. Would any of you like to do that as a training? If yes and if you need help, let me know!

@thea-m I think I did it?

File: OxfordBodleian/Juel-Jensen/BDLaethe33.xml (hunk 2 above)
Comment (MassimoVilla) [reply OxfordBodleian/Juel-Jensen/BDLaethe33.xml:299]: So, do `<seg part="I">` and `<seg part="F">` stand for incipit and explicit respectively? Why "F"?

Comment (thea-m) [reply OxfordBodleian/Juel-Jensen/BDLaethe33.xml]: Something with initial and final, I think...

**Pull request merged**